src/statusBar.ts

```diff
@@ -35,6 +44,11 @@ export class StatusBarManager {
     this.targetItem.show();
   }
 
+  public updateBuildUseCaseItem(useCase: string) {
+    this.buildUseCaseItem.text = `$(gear) L : ${useCase}`;
```

Comment:
Not sure `gear` is the best icon for this, which is a pretty generic setting icon. Maybe we should use `tools` instead ?

![image](https://github.com/LedgerHQ/ledger-vscode-extension/assets/111449109/2b190700-eaf5-443a-8c0e-02deb5000bac)

Reply:
Done